### ISSUE TITLE
Content not responsive

### ISSUE THREAD

**Issue by N6REJ** [gsuez/MasterBootswatch#3]
When viewing blog articles the content is not responsive at all.  The missing view for content causes the layout to be bootstrap 2 and not properly convert to bootstrap 3.
This image shows the horrible layout.
![capture](https://cloud.githubusercontent.com/assets/1850089/7870055/3c7ff2c8-054c-11e5-85a4-0f78defead55.JPG)

**Comment by gsuez** [comment]
you use the last version? send url.

**Comment by N6REJ** [comment]
yes, I'm just downloaded bootswatch a few days ago.  That image is on my localhost so I can't give you a link
in templates/mytemplate/html there is no com_content overrides so of course the layouts will be bootstrap 2

**Comment by gsuez** [comment]
this is features articles menu or blog article menu?

**Comment by gsuez** [comment]
ok, I don´t override any files... if you want, copy from beez3 template, the file html/com_content/category/blog.php file to MB folder.

**Comment by N6REJ** [comment]
featured on the main page.. there are 2 articles I have published to the front page.  the second is the one thats breaking.. to replicate, create 2 articles to front page, leave blog/featured items as default.

**Comment by N6REJ** [comment]
just curiosity why beez 3?

**Comment by gsuez** [comment]
this template use column class, not span class (from bootstrap2). Check it.
